insights/parsers/awx_manage.py

```diff
@@ -59,3 +61,30 @@ class AnsibleTowerLicense(CommandParser, JSONParser):
         29885220
     """
     pass
+
+
+@parser(Specs.awx_manage_print_settings)
+class AwxManagePrintSettings(CommandParser, JSONParser):
+    """
+    The AwxManagePrintSettings class parses the command ``awx-manage print_settings``.
+
+    Sample ``awx-manage print_settings INSIGHTS_TRACKING_STATE SYSTEM_UUID INSTALL_UUID TOWER_URL_BASE AWX_CLEANUP_PATHS AWX_PROOT_BASE_PATH --format json`` file::
```

Comment:
I think it would be clearer here if you just said "Sample command output" instead of including the command in the doc.

Reply:
Updated, please have a check.